pytransform3d/transform_manager.pyi

```diff
@@ -18,20 +20,50 @@ class TransformManager(object):
     predecessors: np.ndarray
     _cached_shortest_paths: Dict[Tuple[int, int], List[Hashable]]
 
-    def __init__(self, strict_check: bool = ..., check: bool = ...): ...
+    def has_frame(self, frame: Hashable) -> bool: ...
 
     def add_transform(self, from_frame: Hashable, to_frame: Hashable,
-                      A2B: np.ndarray) -> "TransformManager": ...
+                      A2B: Any) -> "TransformGraphBase": ...
+
+    def _check_transform(self, A2B: Any) -> Any: ...
+
+    def _recompute_shortest_path(self): ...
 
     def remove_transform(
             self, from_frame: Hashable,
             to_frame: Hashable) -> "TransformManager": ...
 
-    def has_frame(self, frame: Hashable) -> bool: ...
+    def get_transform(
+            self, from_frame: Hashable, to_frame: Hashable) -> Any: ...
```

Comment:
```suggestion
    def get_transform(
            self, from_frame: Hashable, to_frame: Hashable) -> Transform: ...
```

Reply:
We could remove this from the base class if required